### DISCUSSION TITLE
The Bing Map isn't load

### DISCUSSION THREAD

**Discussion by RedElectricity** [community/community#63217] [GitHub Education]
### Hi everyone,

Hi.

I am a senior high school student which in China, I want to regist the package but I am found the Bing Map tiles cannot load, here is the screenshot

<img width="1045" alt="image" src="https://github.com/community/community/assets/75127571/5aa8cdab-8194-4d01-aeaa-641946b4b715">

and the console print
```
j2D80qLgU6kPM4ruVAZWu4t1ObI.br.js:1  Refused to connect to 'https://t0.dynamic.tiles.ditu.live.com/comp/stl?v=8.14&tpp=7DCFF932-3B97-4B3E-9EB4-5B9AA89EFA3C&og=804&ur=CN' because it violates the document's Content Security Policy.
```

### A suggestion...(optional)

Add more map provider and school

**Comment by hanyixuanten** [comment]
same, have you found a solution?

**Comment by Stardito** [comment]
same, CSP error?
